molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/resources/Resources.java

```diff
@@ -24,4 +27,11 @@
 	 */
 	Iterable<Entity> findAll(String name, Query q);
 
+	/**
```

Comment:
ok, well when we needed it, it wasn't in the interface so we had to add it there as well.
It is just to check if the configuration is going smoothly when debugging, there is no other way to get a list of available loaded resource(s) entities. And the errors do not show a clear distinction between missing resources or failed to load files. if the resource file is not available for any reason what so ever it will just try to look the data up in the dataservice. This will then throw an entity not found exception.